src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -70,7 +70,12 @@ export default class ColorPicker extends React.PureComponent<Props, State> {
     ) : null;
 
     return (
-      <div className={styles.ColorPicker} id={id}>
+      <div
+        className={styles.ColorPicker}
+        id={id}
+        onMouseDown={this.handlePickerDrag}
+        onMouseMove={this.handlePickerDrag}
```

Comment:
Do we need the `mousemove` event since we are preventing the default for `mousedown`?

Reply:
Good point! I don't think it's needed, but I will confirm

Reply:
Doesn't seem to be affecting it, removed